src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxKindsService.vb

```diff
@@ -0,0 +1,16 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.LanguageServices
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
+    <ExportLanguageService(GetType(ISyntaxKindsService), LanguageNames.VisualBasic), [Shared]>
+    Friend NotInheritable Class VisualBasicSyntaxKindsService
+        Implements ISyntaxKindsService
+
+        Public ReadOnly Property IfKeyword As Integer = SyntaxKind.IfKeyword Implements ISyntaxKindsService.IfKeyword
+        Public ReadOnly Property LogicalAndExpression As Integer = SyntaxKind.AndAlsoExpression Implements ISyntaxKindsService.LogicalAndExpression
```

Comment:
I thought "logical" was used to imply short-circuiting. If not, what is a good terminology here?

Reply:
I thought `And` would be referred to as a bitwise operator.

Reply:
[It's both!](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/operators/and-operator)

> Performs a logical conjunction on two Boolean expressions, or a bitwise conjunction on two numeric expressions.

Honestly, this might just require a comment.

Reply:
this is the appropriate naming.  We consistently use LogicalAnd to mean 'short-circuiting-`and`' across a lot of the IDE code.

Reply:
Added comment in 1019e00a3575491a0570e9ddb9f50edcb84a3f9e